addons/mail/static/src/components/dialog/dialog.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Dialog" owl="1">
-        <div class="o_Dialog">
+        <div class="o_Dialog" t-att-class="{'o_dialog_mobile': env.messaging.device.isMobile}">
```

Comment:
We've used the modifier `o-mobile` before for this, probably want to keep the convention?

Reply:
yes, and `some_class_name` is the old way of defining classnames, and we don't do that in this refactoring (bad & loose convention to being with).
Any Odoo classnames in discuss refactoring must follow this format:
```
o_ComponentName_partName
o-modifierName
o_ComponentName_partName-modifierName
```